doc/contributor/reference/style-guide.md

```diff
@@ -56,7 +58,7 @@ Use any color from Color Brewer's `YlOrRd09` color palette.
 
 ##### Polygon
 
-Example: "Arctic Protected Areas (CAFF 2017)" for an example.
```

Comment:
This is a good example of redundancy

Reply:
There's a song by Mr. Bungle where the word "redundant" is repeated... redundantly. I think I forgot the name of it ... ;)